### PR TITLE
[Meeting Extensibility]: Content Bubble JavaScript/NodeJS

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
## Description 

Created a Content Bubble Bot in meeting chat having an Adaptive Card sent by Bot , same Adaptive Card displayed in Content Bubble.

**Language Used:** JavaScript/Nodejs

## Testing

Interaction with Content Bubble Bot in Microsoft Teams Meeting chat
1. Bot send an Adaptive Card in meeting conversation and display Content bubble screen in meeting to all the participants
image

Bot redirect user to proper documentation for apps in meeting based on user input

image

### CODE REVIEW

File: samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        await context.sendActivity(context.activity.from.name + " : " +"**"+reply.myValue+"**"+" for " + "'" + reply.title + "'")
+    }
+
+    async contentBubble(context)
+    {
+      const replyActivity = MessageFactory.text("**Please provide your valuable feedback**");
+      replyActivity.channelData = {
+        notification: {
+          alertInMeeting: true,
+          externalResourceUrl: 'https://teams.microsoft.com/l/bubble/<<APP_ID>>?url=<<ENDPOINT_URL>>&height=270&width=300&title=ContentBubbleinTeams&completionBotId=<<APP_ID>>'
```
Comment (Wajeed-msft) [inline samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js:45]: ReadMe file does not talk about setting these values, could you please update this?

File: samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    createAdaptiveCard(){
+      return CardFactory.adaptiveCard(AdaptiveCard);
+    }
+
+    createQuestionAdaptiveCard(myText){
+    var templatePayload = {
+      "type": "AdaptiveCard",
+      "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+      "version": "1.0",
```
Comment (Wajeed-msft) [inline samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js:59]: Could you please move the template JSON to separate file?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js (hunk 2 above)
Comment (Trinetra-MSFT) [reply samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js]: Added QuestionTemplate.json in resources folder and updated the code to use that

File: samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js (hunk 1 above)
Comment (Trinetra-MSFT) [reply samples/meetings-content-bubble/nodejs/Content Bubble  Nodejs/bots/contentBubbleBot.js:46]: I've added one more point to set the externalResourceUrl in content bubble notification payload

File: samples/meetings-content-bubble/nodejs/bots/contentBubbleBot.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        await context.sendActivity(context.activity.from.name + " : " +"**"+reply.myValue+"**"+" for " + "'" + reply.title + "'")
+    }
+
+    async contentBubble(context)
+    {
+      const replyActivity = MessageFactory.text("**Please provide your valuable feedback**");
+      replyActivity.channelData = {
+        notification: {
+          alertInMeeting: true,
+          externalResourceUrl: 'https://teams.microsoft.com/l/bubble/<<APP_ID>>?url=<<ENDPOINT_URL>>&height=270&width=300&title=ContentBubbleinTeams&completionBotId=<<APP_ID>>'
```
Comment (Wajeed-msft) [inline samples/meetings-content-bubble/nodejs/bots/contentBubbleBot.js:46]: Can we update this to pull the values from env file?

File: samples/meetings-content-bubble/nodejs/teamsAppManifest/manifest.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
```
Comment (Wajeed-msft) [inline samples/meetings-content-bubble/nodejs/teamsAppManifest/manifest.json:2]: Could you please update this to use latest manifest schema version?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by Trinetra-MSFT** [comment]
@Wajeed-msft , Updated the Review comments please check

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**